src/web.ts

```diff
@@ -4,7 +4,7 @@ import { ApplicationFunctionOptions, Probot, Server } from "probot";
 
 import { Config, loadConfig } from "./config";
 import { Database, PRAction } from "./db";
-import { createFailureIssue, getRequiredChecks, mergePR } from "./github";
+import { addComment, appendPRDescription, createFailureIssue, getRequiredChecks, mergePR } from "./github";
```

Comment:
Nit: does `addComment` still need to be imported? I guess I'm surprised a linter or something didn't get upset at this.

Reply:
Good catch. I noticed packages get added automatically but not removed. I might be missing some tooling around TypeScript...